sdk/ai/azure-ai-projects/README.md

```diff
@@ -718,6 +718,122 @@ print("Deleted agent")
 
 <!-- END SNIPPET -->
 
+### Evaluation
+Evaluation in Azure AI Project client library is desinged to assess the performance of generative AI applications in cloud. Generative AI application generations are quantitatively measured with mathematical based metrics, AI-assisted quality and safety metrics. Metrics are defined as evaluators. Built-in or custom evaluators can provide comprehensive insights into the application's capabilities and limitations.
+
+#### Evaluator
+Evaluators are custom or prebuilt classes or functions that are designed to measure the quality of the outputs from language models or generative AI applications.
+
+Evaluators are made available via [azure-ai-evalaution][azure_ai_evaluation] SDK for local experience and also in [Evaluator Library][evaluator_library] in Azure AI Studio for using them in the cloud.
+
+More details on built-in and custom evaluators can be found [here][evaluators].
+
+#### Run Evalaution in cloud:
+
+To run evaluation in cloud following are needed:
+
+- Evaluators
+- Data to be evaluated
+- [Optional] Azure Open AI model.
+
+##### Evaluators
+
+For running evaluator in cloud, evaluator id is needed. To get via code you use [azure-ai-evaluation][azure_ai_evaluation]
+
+```
+# pip install azure-ai-evaluation
+
+from azure.ai.evaluation import RelevanceEvalautor
+
+evaluator_id = RelevanceEvaluator.id
+```
+
+##### Data to be evaluated
+Evaluation in cloud supports data in form of `jsonl` file. Data can be uploaded via the helper method `upload_file` on the project client.
+
+```python
+# Creating project client
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+
```

Comment:
We have one place at the top where we define project_client. All samples can assume project_client is already defined, and no need to show it again.